### PR TITLE
Change query parameter names to bring into line with other ogc api implementations

### PR DESCRIPTION
**Pull request by m-burgoyne**
Documentation updated as discussed in #96, `time` query parameter changed to `datetime` and  `outputformat` query parameter changed to `f` to bring the examples into line with the approach taken by other OGC API implementations.

### CODE REVIEW

File: candidate-standard/requirements/edr/REQ_rc-interpolation-response.adoc [hunk 1]
```diff
@@ -9,6 +9,6 @@
 interpolation = interpolation-method 
 ----
 ^|C|If not specified the query will use the default interpolation method for the chosen collection.
-^|D|If the interpolation method value e does not match one of the listed methods a 403 error message should be returned.
+^|D|If the interpolation method value e does not match one of the listed methods a 400 error message should be returned.
```
Comment (dblodgett-usgs) [inline candidate-standard/requirements/edr/REQ_rc-interpolation-response.adoc:12]: tics?

File: candidate-standard/requirements/edr/REQ_rc-interpolationHeight-response.adoc [hunk 2]
```diff
@@ -10,5 +10,5 @@
 interpolationz = "R"interval "/" interpolation-method
 ---- 
 ^|C |The coordinates of the `coords` parameter define the centre point of the corridor. 
-^|D |If the interpolation method value does not match one of the listed methods a 403 error message should be returned.
+^|D |If the interpolation method value does not match one of the listed methods a 400 error message should be returned.
```
Comment (dblodgett-usgs) [inline candidate-standard/requirements/edr/REQ_rc-interpolationHeight-response.adoc:13]: tics?

File: candidate-standard/requirements/edr/REQ_rc-outputFormat-definition.adoc [hunk 3]
```diff
@@ -2,11 +2,11 @@
 [width="90%",cols="2,6a"]
 |===
 |*Requirement {counter:req-id}* | */req/edr/outputFormat-definition* 
-^|A |Each resource collection operation SHALL support a parameter `outputformat`with the following characteristics (using an OpenAPI Specification 3.0 fragment):
+^|A |Each resource collection operation SHALL support a parameter `f`with the following characteristics (using an OpenAPI Specification 3.0 fragment):
```
Comment (dblodgett-usgs) [inline candidate-standard/requirements/edr/REQ_rc-outputFormat-definition.adoc:5]: missing space after `f`

File: candidate-standard/requirements/edr/REQ_rc-outputCRS-response.adoc [hunk 4]
```diff
@@ -4,5 +4,5 @@
 |*Requirement {counter:req-id}* | */req/edr/outputCRS-response*
 ^|A|If the `crs` parameter is provided, the returned information should be reprojected  (if required) to the defined coordinate system.  If the `crs` parameter is not specifed the data will be returned in its native projection.
 ^|B|The `crs` parameter SHALL consist of an identifier selected from the enumerated list of valid values supplied in the collections metadata.
-^|C| if an unsupported `crs` value is requested a `403` error message SHOULD be returned .
+^|C| if an unsupported `crs` value is requested a 400 error message SHOULD be returned .
```
Comment (dblodgett-usgs) [inline candidate-standard/requirements/edr/REQ_rc-outputCRS-response.adoc:7]: tics are left in and taken out variously?

File: candidate-standard/requirements/edr/REQ_rc-interpolation-response.adoc (hunk 1 above)
Comment (chris-little) [reply candidate-standard/requirements/edr/REQ_rc-interpolation-response.adoc]: Mark and I were out of phase

File: candidate-standard/requirements/edr/REQ_rc-outputCRS-response.adoc (hunk 4 above)
Comment (chris-little) [reply candidate-standard/requirements/edr/REQ_rc-outputCRS-response.adoc]: @dblodgett-usgs No, I just had a stint earlier sweeping up the missing ticks from my PR

**Pull request merged**